weather.go

```diff
@@ -0,0 +1,111 @@
+package weather
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type apiResponse struct {
+	Name    string
+	Weather []struct {
+		Main string
+	}
+	Main struct {
+		Temp float64
+	}
+}
+
+type Conditions struct {
+	City               string
+	OneWord            string
+	TemperatureCelsius float64
+}
+
+func RunCLI(args []string) {
+
+	token := os.Getenv("WEATHERAPP_TOKEN")
+	if token == "" {
+		fmt.Fprintf(os.Stderr, "please set env variable, WEATHERAPP_TOKEN \n")
+		os.Exit(2)
+	}
+
+	request, err := Request(args, token)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "problem setting url', %v", err)
```

Comment:
What would the error be here? Only a missing location, right? So we can be more helpful here:

```suggestion
		fmt.Fprintf(os.Stderr, "Usage: %s LOCATION\n", os.Args[0])
```